samples/dotnet-isolated/eventhub/KafkaOutput.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.Logging;
+using Microsoft.Azure.Functions.Worker.Http;
+using System.Net;
+
+namespace Eventhub
+{
+    public class KafkaOutput
+    {
+        [Function("KafkaOutput")]
+        
+        public static MultipleOutputType Output(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = null)] HttpRequestData req,
+            FunctionContext executionContext)
+        {
+            var log = executionContext.GetLogger("HttpFunction");
+            log.LogInformation("C# HTTP trigger function processed a request.");
+
+            string message = req.FunctionContext
+                                .BindingContext
+                                .BindingData["message"]
+                                .ToString();
+
+            var response = req.CreateResponse(HttpStatusCode.OK);
+            return new MultipleOutputType()
+            {
+                Kevent = message,
+                HttpResponse = response
+            };
+        }
+    }
+
+    public class MultipleOutputType
+    {
+        [KafkaOutput("BrokerList",
+                    "topic",
+                    Username = "$ConnectionString",
+                    Password = "EventHubConnectionString",
```

Comment:
Should we add a commit to explain the difference between $ConnectionString and EventhubConnectionString?

Reply:
Added it in Readme.